BootStrapPartitionsTest is unstable

Stacktrace:

```
[ERROR] Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.88 s <<< FAILURE! - in io.zeebe.broker.it.startup.BootstrapPartitionsTest
[ERROR] shouldCreateDefaultPartitionsOnBootstrappedBrokers(io.zeebe.broker.it.startup.BootstrapPartitionsTest)  Time elapsed: 0.869 s  <<< FAILURE!
java.lang.AssertionError:

Actual and expected have the same elements but not in the same order, at index 0 actual element was:
  <1>
whereas expected element was:
  <0>

        at io.zeebe.broker.it.startup.BootstrapPartitionsTest.shouldCreateDefaultPartitionsOnBootstrappedBrokers(BootstrapPartitionsTest.java:45)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
        at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
        at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
        at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
        at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)
        at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)
        at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
        at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.lang.Thread.run(Thread.java:748)
```

--- a/qa/integration-tests/src/test/java/io/zeebe/broker/it/startup/BootstrapPartitionsTest.java
+++ b/qa/integration-tests/src/test/java/io/zeebe/broker/it/startup/BootstrapPartitionsTest.java
@@ -42,6 +42,6 @@ public class BootstrapPartitionsTest {
 
     // then
     assertThat(partitions.size()).isEqualTo(2);
-    assertThat(partitions).containsExactly(0, 1);
+    assertThat(partitions).containsExactlyInAnyOrder(0, 1);
   }
 }

